hw4/03-form.js

```diff
@@ -1,16 +1,47 @@
 const express = require('express');
+const path = require('path');
 const app = express();
 const port = process.env.PORT || 5001;
 
 // Use middleware static() to serve all static files in the given folder
 app.use(express.static('public'));
+app.use('/index', express.static('public/files'));
 
 // Use middleware urlencoded() to parse an incoming request with a urlencoded payload and return an objectß
-app.use(express.urlencoded({ extended: false }));
+
+app.use(
+  express.urlencoded({
+    extended: false,
+  })
+);
+
+
+
+app.get('/form', (req, res) => {
+  res.sendFile(path.join(__dirname + '/public/index.html'));
+  res.end()
+});
 
 // POST request
-app.post('', (req, res) => {
+app.post('/submit', (req, res) => {
   // Add your code here
+  res.writeHead(200, { 'Content-Type': 'text/html' });
+  let toText = req.body.message;
+  const check = req.body.news;
+  let toPrint = '';
+  if (toText === '') {
```

Comment:
This is something I forgot to do in my exercise. I remember the style guide mentions that we should use explicit comparison for strings.